grc/main.py

```diff
@@ -201,7 +202,9 @@ def get_config_file_path(config_file: str = "grc.conf") -> str:
             return oldpath
         # Default to the correct path if both are configured.
         # neither old, nor new path exist: create new path, return that
-        os.makedirs(newpath, exist_ok=True)
+        path_parts = Path(newpath).parts[:-1]
+        pathdir = os.path.join(*path_parts)
+        os.makedirs(pathdir, exist_ok=True)
```

Comment:
needs to be applied to the `os.makedirs(xdgcand…` below, as well. Otherwise: :+1: 

Sorry that I introduced this bug!